helm/panda/charts/jedi/sandbox/init-jedi

```diff
@@ -2,15 +2,14 @@
 /opt/panda/bin/panda_common-install_igtf_ca > /var/log/panda/install_igtf_ca.log 2>&1
 
 # fetch info from CRIC before running services
-tmpExe=`mktemp`
-cat <<EOT >> ${tmpExe}
-/opt/panda/usr/bin/panda_server-schedconfig_json
-/opt/panda/usr/bin/panda_server-configurator
-EOT
-chmod +x ${tmpExe}
-
 if [[ -z "${PANDA_NO_ROOT}" ]]; then
-  runuser -u atlpan -g zp -- ${tmpExe}
+  runuser -u atlpan -g zp -- /opt/panda/usr/bin/panda_server-schedconfig_json
+  runuser -u atlpan -g zp -- /opt/panda/usr/bin/panda_server-configurator
 else
-  ${tmpExe}
+  /opt/panda/usr/bin/panda_server-schedconfig_json
+  /opt/panda/usr/bin/panda_server-configurator
 fi
+
+# setup msg
```

Comment:
Those lines are not required since symlinks are in the container

https://github.com/PanDAWMS/panda-jedi/blob/master/Dockerfile#L40-L41

and *.json in the sandbox dir are copied to /data/panda. 
You need to specify

/opt/panda/etc/panda/jedi_msg_proc_config.json

at
https://github.com/PanDAWMS/panda-k8s/blob/89447d5e4a769ab548c71255d037bd871e4eec4a/helm/panda/charts/jedi/panda_jedi_config.json#L13

and

/opt/panda/etc/panda/jedi_mq_config.json 

at https://github.com/PanDAWMS/panda-k8s/blob/89447d5e4a769ab548c71255d037bd871e4eec4a/helm/panda/charts/jedi/panda_jedi_config.json#L16

Reply:
updated.